rke/templates/flannel_v0.19.2.go

```diff
@@ -174,16 +134,7 @@ spec:
 {{end}}
       hostNetwork: true
 # Rancher specific change
-{{- if .KubeFlannelPriorityClassName }}
-      priorityClassName: {{ .KubeFlannelPriorityClassName }}
-{{- end }}
-{{if .NodeSelector}}
```

Comment:
We should carry forward {{.NodeSelector}} changes, it's a rancher specific change and comes from rke.

Reply:
The upstream chart adopts the `{{.NodeSelector}` section, so I removed this section to reduce duplicates.  

see line 169-174